dev_envs/mac-env-setup.md

```diff
@@ -45,7 +45,10 @@ xcode-select --install
 
 ### Docker ###
 
-Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
+> **Note**
+> *[Colima](https://github.com/abiosoft/colima) is our preferred solution on macOS.*
```

Comment:
I think this renders a little better, and note that I changed the Colima link to point to the relevant section of this document:
```suggestion
> [!NOTE]
> [Colima](#colima-a-docker-alternative) is our preferred solution on macOS.
```